src/fields/u32/fp/wrapper.rs

```diff
@@ -99,3 +177,19 @@ impl Neg for Fp {
         Fp(result)
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[test]
+    fn inversion_test() {
```

Comment:
It would be cool if these were proptests i.e. something like:

```Rust

    fn fp_strategy() -> BoxedStrategy<Fp> {
        any::<[u8; 48]>()
            .prop_map(|bytes| Fp::from_bytes(&bytes))
            .boxed()
    }

    proptest! {
        #[test]
        fn inversion_proptest(element in fp_strategy().prop_filter("Non-zero element", |e| e != &Fp::zero())) {
            let inverse = element.inverse().unwrap();
            assert_eq!(element.mul(inverse), Fp::one());
        }
    }
```

Reply:
Good point. In #67, we implement a proptest for inversion, but I'll include this test as well.